mypy/build.py

```diff
@@ -797,9 +801,59 @@ def is_file(path: str) -> bool:
     return res
 
 
-def find_module(id: str, lib_path_arg: Iterable[str]) -> Optional[str]:
+SITE_PACKAGE_COMMANDS = (
+    # User site packages
+    '"import site;print(site.getusersitepackages());print(*site.getsitepackages(), sep=\'\\n\')"',
+    # for virtualenvs
+    '"from distutils.sysconfig import get_python_lib;print(get_python_lib())"',
+)
+
+
+def call_python(python: str, command: str) -> str:
+    return check_output(python + ' -c ' + command,
+                        stderr=STDOUT).decode('UTF-8')
```

Comment:
Why do you need to redirect stderr? And is utf-8 safe? Maybe we should pass PYTHONIOENCODING (https://docs.python.org/3/using/cmdline.html#envvar-PYTHONIOENCODING).

Reply:
I originally wanted to handle error text, but I think it may not be needed after all.

Reply:
I'm not the most familiar with best practices for encoding/decoding, but wouldn't the encoding of `sys.stdout` be most proper?